README.md

```diff
@@ -123,6 +123,41 @@ You can look at them on the commandline with:
 ./start-test-env spawn-test-environment --help 
 ```
 
+### The integration-test-docker-environment command line tool
```

Comment:
ok, that looks good. However, please create a ticket, that we move things from the README to the User Guide.

Reply:
:+1: done see #185